include/ps/internal/customer.h

```diff
@@ -12,7 +12,7 @@
 #include <thread>
 #include <memory>
 #include "ps/internal/message.h"
-#include "ps/internal/threadsafe_queue.h"
+#include "ps/internal/threadsafe_pqueue.h"
 namespace ps {
```

Comment:
Could we also have an MXNet PR that update the 3rdparty dependency to `anandj91:prio` temporarily to verify that all mxnet kvstore test cases pass?

Reply:
https://github.com/apache/incubator-mxnet/pull/15124
I have raised a PR on MXNet. There are some test cases failing now. Some of them are failing because of build issues. I'll take a look into it and fix it soon.